nuitka/plugins/standard/standard.nuitka-package.config.yml

```diff
@@ -2920,20 +2920,21 @@
   implicit-imports:
     - post-import-code:
         - |
-          def myconnect(self, func):
+          def patched_connect(self, func, conn_type=None):
+              conn_type = conn_type or QtCore.Qt.ConnectionType.AutoConnection
```

Comment:
Seems I failed to check the signature. Generally it could be better to just use `*args, **kwargs` all the time, but I don't know how to properly get good errors out of that. Getting a `patched_connect` to fail, is going to make things more clear to victims of it.